frontend/src/components/Form.test.js

```diff
@@ -45,3 +46,67 @@ describe('Form', () => {
     );
   });
 });
+
+describe('Form Validation', () => {
+  it('should return true', () => {
+    const role = 'Intern';
+    const productArea = 'Platforms and Ecosystems';
+    // Wed Jul 15 2020 20:56:45
+    const timeAvailableUntilMilliseconds = 1594846605591;
+    const duration = 15;
+    // Wed Jul 15 2020 08:30:00
+    const currentTimeMilliseconds = 1594801800000;
+    expect(validateFormInputs(role, productArea, duration,
+        timeAvailableUntilMilliseconds, currentTimeMilliseconds)).toBe(true);
+  });
+
+  it('should return false since Role is not specified', () => {
```

Comment:
tiny nit, here and below s/since/when

Reply:
Changed to "should return ____ when..." format